packages/p2p-v2/src/pages/orders/components/ChatMessageText/ChatMessageText.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { memo, PropsWithChildren } from 'react';
+import { Text, useDevice } from '@deriv-com/ui';
+import './ChatMessageText.scss';
+
+type TChatMessageTextProps = {
+    color: string;
+    type?: string;
+};
+
+const ChatMessageText = ({ children, color, type = '' }: PropsWithChildren<TChatMessageTextProps>) => {
+    const { isMobile } = useDevice();
+    return (
+        <div className='p2p-v2-chat-message-text'>
+            <Text color={color} lineHeight='xl' size={type === 'admin' && !isMobile ? 'xs' : 'sm'}>
```

Comment:
@nada-deriv, if the `xs` is for desktop, why not use `isDesktop` instead? 🤔 
```suggestion
            <Text color={color} lineHeight='xl' size={type === 'admin' && isDesktop ? 'xs' : 'sm'}>
```